_includes/charts/tigera-operator/templates/tigera-operator/02-role-tigera-operator.yaml

```diff
@@ -124,6 +175,17 @@ rules:
       - networksets
     verbs:
       - create
+  # For AWS security group setup.
+  - apiGroups:
+      - batch
+    resources:
+      - jobs
+      - cronjobs
+    verbs:
+      - create
+      - update
+      - list
+      - watch
```

Comment:
I see jobs and cronjobs only created for openshift. I think they should be present always for IDS installer job and curator cronjob.